### DISCUSSION TITLE
Allow to remove organization from the webpage

### DISCUSSION THREAD

**Discussion by dzek69** [npm/feedback#559] [Website] [answered]
I created organization by accident and to delete it I needed to contact support. I actually did it again today.

I think we should be able  to do that from the web panel if there are no packages published yet. I suspect that many people will just ignore the empty organizations they created but there could be nice names taken because of that.

**Comment by MylesBorins** [comment, marked as answer]
@dzek69 if you click billing on an org there is an option to delete the org yourself

https://docs.npmjs.com/deleting-an-organization
